site/source/docs/tools_reference/emcc.rst

```diff
@@ -282,6 +282,9 @@ Options that are modified or new in *emcc* are listed below:
 ``--exclude-file <name>``
 	Files and directories to be excluded from :ref:`--embed-file <emcc-embed-file>` and :ref:`--preload-file <emcc-preload-file>`. Wildcards (*) are supported.
 
+``--fully-prepare-files``
+	Tells the file packager to not only load the preloaded files, but also prepare them, similarly to calling ``emscripten_async_prepare()``: This also does preparation tasks like decoding images using the browser codecs.
```

Comment:
`--use-preload-plugins` sounds quite good.

Renaming `emscripten_async_prepare` would break existing client code - is that a concern? Or do you mean we would retain `emscripten_async_prepare` as a variant without the preload decoding, and then have the `emscripten_async_prepare_with_preload_plugins` as a separate one that does preload decoding as well?